.gitignore

```diff
@@ -101,3 +101,4 @@ gmailLogin.properties
 webLogin.properties
 mailHandlerTestProps.properties
 config.yaml
+mturk.properties
```

Comment:
because this is completly undepding from each other - the main config maybe does not configure a mturk platform, maybe you want to use other credetials for the test etc. 

And its actual 2 files.